test/case/convention_testing/openshift/golang-osd-cluster-operator/01_generated_files_checker

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+REPO_ROOT=$(git rev-parse --show-toplevel)
+
+source $REPO_ROOT/test/lib.sh
+
+repo=$(empty_repo)
+add_cleanup $repo
+test_project="file_generator"
+
+# TODO: test REPO_NAME detection by setting an appropriately-formatted
+# git remote
+export REPO_NAME=${0##*/}
+
+bootstrap_project $repo ${test_project} openshift/golang-osd-cluster-operator
+cd $repo
+export GOPATH=$GOPATH:`pwd`
+
+make update_boilerplate
+make check_generation
+if [ $? = 0 ] ; then 
+	echo "No error detected while checking on uncommitted project"
+	exit 1
+else
+	echo "Project is not committed. Checks failed as expected"
+fi
+
+git add -A
+git commit -m "initial commit"
+
+make check_generation
```

Comment:
Not a comment, more of a question. I don't understand why things should fail here. Because the modified changes are just been committed in the previous line.

Reply:
The project initially does not contain the generated files so after the check_generation, as it trigger the code generation, there is a new diff.

Reply:
I would recommend that the test for the gogenerate not only test that the file has been generated but also that the contents of the generated file match the expected generated out. Also a test case can be added where the contents of the generated file are modified and detected.

Reply:
Yes, that's planned once the 'simple' case is working.